src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jobs/oneoff/linksupersededappointments/LinkSupersededAppointmentsJobConfiguration.kt

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jobs.oneoff.linksupersededappointments
```

Comment:
@awhodgson256  This file should be deleted in the revert but it is shown as a new file?

Reply:
@kth13 I think both of these issues are due to reverting back to the state of the first merge (IE yesterday morning).  As I understand it this will allow us to do a revert on yesterday's first merge without conflicts.